web/src/components/storage/SpaceActionsTable.tsx

```diff
@@ -43,8 +43,25 @@ import {
 } from "~/components/storage/device-utils";
 import { Icon } from "~/components/layout";
 import { PartitionSlot, SpacePolicyAction, StorageDevice } from "~/types/storage";
+import { configModel } from "~/api/storage/types";
 import { TreeTableColumn } from "~/components/core/TreeTable";
 import { Table, Td, Th, Tr, Thead, Tbody } from "@patternfly/react-table";
+import { useConfigModel } from "~/queries/storage/config-model";
+
+function isUsedPartition(partition: configModel.Partition): boolean {
```

Comment:
Just a note because I do not remember if I already commented it in a PR from you: until now we were using the fat arrow sintax 🤷‍♂️